net/aria2/files/aria2.init

```diff
@@ -154,23 +154,23 @@ aria2_validate() {
 
 aria2_start() {
 	local section="$1"
-	aria2_validate "$section" || { _err "Validation failed."; return 1; }
+	[ "$2" = 0 ] || { _err "Validation failed."; return 1; }
```

Comment:
add quotes to zero?

Reply:
The quotes are not part of the (string) comparison; they are consumed by the shell. There are quotes around `$2` because there is a (remote) possibility that it contains spaces, and quoting it ensures it is passed as one argument to `[` (`test`).

Reply:
I know quotes are not necessary in this condition, add quotes to make it unify. (everywhere else in the script added.)